README.md

```diff
@@ -9,7 +9,7 @@ We are introducing a new version of the Sendbird Chat UIKit. Version 3 features
 
 ## Installation
 
-`npm i @sendbird/uikit-react`
```

Comment:
Dont need to remove this, clients can use any pkg manager ~ is better to keep both :) 

`npm i @sendbird/uikit-react`
or
`yarn add @sendbird/uikit-react `

Reply:
Ahh yeah I thought about this too. Will describe both then :)

Reply:
https://github.com/sendbird/sendbird-uikit-react/pull/473/commits/17e04b7e383044976bfdac83ff1c72248029c863 will this be okay? Feel free to re-commit if you want to touch more